app/styles/core/_form.less

```diff
@@ -0,0 +1,6 @@
+.oss-label {
+    font-size: var(--font-size-md);
+    color: var(--color-gray-500);
+    font-weight: 400;
```

Comment:
Pretty sure this isn't respecting our 2 spaces indentation
You can compare with other files and probably set your IDE to match the repo settings (if any... ^^)